src/Common/StackTrace.cpp

```diff
@@ -230,26 +238,41 @@ void StackTrace::symbolize(
                 auto dwarf_it = dwarfs.try_emplace(object->name, object->elf).first;
 
                 DB::Dwarf::LocationInfo location;
-                std::vector<DB::Dwarf::SymbolizedFrame> inline_frames;
                 if (dwarf_it->second.findAddress(
-                        uintptr_t(current_frame.physical_addr), location, DB::Dwarf::LocationInfoMode::FAST, inline_frames))
+                        uintptr_t(current_frame.physical_addr), location, mode, inline_frames))
                 {
                     current_frame.file = location.file.toString();
                     current_frame.line = location.line;
                 }
             }
         }
-        else
-            current_frame.object = "?";
 
         if (const auto * symbol = symbol_index.findSymbol(current_frame.virtual_addr))
             current_frame.symbol = demangle(symbol->name);
-        else
-            current_frame.symbol = "?";
+
+        for (const auto & frame : inline_frames)
+        {
+            StackTrace::Frame current_inline_frame;
+            const String file_for_inline_frame = frame.location.file.toString();
+
+            current_inline_frame.file = "inlined from " + file_for_inline_frame;
+            current_inline_frame.line = frame.location.line;
+            current_inline_frame.symbol = frame.name;
+
+            callback(current_inline_frame);
+        }
+
+        callback(current_frame);
     }
 #else
-    for (size_t i = 0; i < size; ++i)
-        frames[i].virtual_addr = frame_pointers[i];
+    (void)fatal;
```

Comment:
`[[maybe_unused]]`?

Reply:
OK, rebased.

Reply:
I want this to be conditional, i.e. to suppress it only for non-ELF (to ensure that the value is used), that's why I've converted it to the `(void)x` instead of `[[maybe_unused]]`, but `[[maybe_unused]]` does not work for statements:

```
Jan 11 05:58:25 /build/src/Common/StackTrace.cpp:268:7: error: 'maybe_unused' attribute cannot be applied to a statement
Jan 11 05:58:25   268 |     [[maybe_unused]] fatal;
Jan 11 05:58:25       |       ^              ~~~~~
Jan 11 05:58:25 /build/src/Common/StackTrace.cpp:268:22: error: expression result unused [-Werror,-Wunused-value]
Jan 11 05:58:25   268 |     [[maybe_unused]] fatal;
Jan 11 05:58:25       |                      ^~~~~

```

Reply:
I've converted it to `UNUSED` macro.